reporters/reporter-default/src/main/java/org/radargun/reporting/commons/TestAggregations.java

```diff
@@ -97,23 +99,41 @@ private void addIteration(Report.Test test, List<Aggregation> iterations, Report
       if (!totalStats.isPresent()) {
          log.warn("There are no stats for this iteration");
       } else {
+         if (test.getGroupOperationsMap() != null) {
+            for (Map.Entry<String, Set<Operation>> op : test.getGroupOperationsMap().entrySet()) {
+               //register groups for thread statistics
+               totalStats.get().registerOperationsGroup(op.getKey(), op.getValue());
+               //register groups for node statistics
+               nodeStats.stream().forEach(ns -> ns.registerOperationsGroup(op.getKey(), op.getValue()));
+               //register groups for thread statistics
+               for (Map.Entry<Integer, List<Statistics>> mapEntry : it.getStatistics()) {
+                  for (Statistics ts : mapEntry.getValue()) {
+                     ts.registerOperationsGroup(op.getKey(), op.getValue());
+                  }
+               }
+            }
+         }
+
          iterations.add(new Aggregation(nodeStats, nodeThreads, totalStats.get(), totalThreads.get(), it));
          for (String operation : totalStats.get().getOperations()) {
             DefaultOutcome defaultOutcome = totalStats.get().getRepresentation(operation, DefaultOutcome.class);
-            if (defaultOutcome == null || defaultOutcome.requests > 0) operations.add(operation);
+            if (defaultOutcome == null || defaultOutcome.requests > 0)
+               operations.add(operation);
```

Comment:
If you're going to refactor stuff like this, I'd add braces, just moving it to a new line feels even more error-prone...